### PR TITLE
Update sap-successfactors-connector-reference.adoc

### PR DESCRIPTION
**Pull request by valean27**

### CODE REVIEW

File: sap-successfactors/4.0/modules/ROOT/pages/sap-successfactors-connector-reference.adoc [hunk 1]
```diff
@@ -27,6 +27,10 @@ Anypoint Connector for SAP SuccessFactors (SuccessFactors Connector) provides fu
 [[config_basic]]
 == Basic Authentication 
 
+=== Note
+
+Basic Authentication requires Minimum Mule EE Version 4.2.0.
```
Comment (valkyrie69) [inline sap-successfactors/4.0/modules/ROOT/pages/sap-successfactors-connector-reference.adoc]: Mule 4.2.0 and later is required for Basic authentication.

File: sap-successfactors/4.0/modules/ROOT/pages/sap-successfactors-connector-reference.adoc [hunk 2]
```diff
@@ -74,6 +78,10 @@ Anypoint Connector for SAP SuccessFactors (SuccessFactors Connector) provides fu
 [[Config_Oauth]]
 == OAuth SAML Bearer Assertion Authentication
 
+=== Note
+
+OAuth SAML Bearer Assertion Authentication requires Minimum Mule EE Version 4.2.2.
```
Comment (valkyrie69) [inline sap-successfactors/4.0/modules/ROOT/pages/sap-successfactors-connector-reference.adoc]: Mule 4.2.2 and later is required for OAuth SAML bearer assertion authentication.

File: sap-successfactors/4.0/modules/ROOT/pages/sap-successfactors-connector-reference.adoc (hunk 1 above)
Comment (sorinmare) [reply sap-successfactors/4.0/modules/ROOT/pages/sap-successfactors-connector-reference.adoc]: This is more accurate

File: sap-successfactors/4.0/modules/ROOT/pages/sap-successfactors-connector-reference.adoc (hunk 2 above)
Comment (sorinmare) [reply sap-successfactors/4.0/modules/ROOT/pages/sap-successfactors-connector-reference.adoc]: This is more accurate

**Review by valkyrie69 — APPROVED** [review]

**Pull request merged**